scripts/rag/llama_index_w_evals_and_qa.py

```diff
@@ -504,7 +548,7 @@ def main():
 
 if __name__ == "__main__":
     program_start = time.time()
-    main()
+    asyncio.run(main())
```

Comment:
@anticorrelator this did make the code run to completion - I think it has something to do with running llm_classify multiple times without awaiting. Left some notes in #2035